.github/workflows/tests.yaml

```diff
@@ -107,11 +70,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kind-k8s-version: [1.21.14, 1.22.15, 1.23.13, 1.24.7, 1.25.3]
+        kind-k8s-version: [1.22.17, 1.23.17, 1.24.12, 1.25.8, 1.26.3]
```

Comment:
Are we deprecating support for 1.21? Do we need to document that, or that we won't test against it anymore?

Reply:
Good point, it's no longer supported by AKS, EKS, GKE, or the Kubernetes team, so I think we should drop it. I'll follow up with a docs PR for that.

Reply:
Opened https://github.com/hashicorp/vault/pull/20010